Mathlib/Algebra/Module/Submodule/Basic.lean

```diff
@@ -592,10 +587,7 @@ theorem sub_mem_iff_right (hx : x ∈ p) : x - y ∈ p ↔ y ∈ p := by
 #align submodule.sub_mem_iff_right Submodule.sub_mem_iff_right
 
 instance addCommGroup : AddCommGroup p :=
```

Comment:
ditto

Reply:
Trying this out

Reply:
These stressed the elaborator in a couple of places but otherwise seem fine to me.